### ISSUE TITLE
Plugin gets confused when using multiple tabs

### ISSUE THREAD

**Issue by timtyrrell** [shortcuts/no-neck-pain.nvim#268]
## Description

Plugin gets confused after using it in one tab then moving to another tab

## Steps to reproduce

1. open a file in nvim
2. open second tab with another file in it
3. in the second tab, run `:NoNeckPain` twice (to shrink and re-expand the screen)
4. switch to the first tab, run `:NoNeckPain` and you are sent to the second tab without anything being resized.
5. you can continue to rerun step 4 with the same outcome

## Expected behavior

No matter which tab you are currently on, `:NoNeckPain` will work as intended

## Environment

- Neovim version:
```
:version
NVIM v0.10.0-dev-3969+g5e5f5174e-Homebrew
Build type: Release
LuaJIT 2.1.1700008891
Run ":verbose version" for more info
```
- no-neck-pain.nvim version "latest"

my `mini.vim`
```
set nocompatible
filetype plugin indent on
syntax on
set hidden

call plug#begin('~/.config/nvim/plugged')

Plug 'shortcuts/no-neck-pain.nvim'

call plug#end()

lua << EOF

require("no-neck-pain").setup({
    width = 100,
    buffers = {
        colors = {
            background = "tokyonight-moon",
        }
    },
    mappings = {
        enabled = true,
        toggle = "<Leader>nn",
        toggleLeftSide = "<Leader>nql",
        toggleRightSide = "<Leader>nqr",
        widthUp = { mapping = "<Leader>n=", value = 20 },
        widthDown = { mapping = "<Leader>n-", value = 20 },
        scratchPad = "<Leader>ns",
    },
})

EOF
```

**Comment by shortcuts** [comment]
hey, the fix is available in [the 1.6.1 version](https://github.com/shortcuts/no-neck-pain.nvim/releases/tag/v1.6.1)

**Comment by timtyrrell** [comment]
I am still seeing it, I do have the latest

>>>
❯ cd ~/.config/nvim/plugged
❯ cd no-neck-pain.nvim
❯ git log -1
commit 28ef4de7ac1e6554be2ef5a72a200584eefe5df5 (HEAD -> main, tag: v1.6.1, origin/main, origin/HEAD)
Author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Date:   Tue Nov 28 22:30:12 2023 +0100

    chore(main): release 1.6.1 (#270)

    :robot: I have created a release *beep* *boop*
    ---


    ##
    [1.6.1](https://github.com/shortcuts/no-neck-pain.nvim/compare/v1.6.0...v1.6.1)
    (2023-11-28)


    ### Bug Fixes

    * **tabs:** prevent unsynced state in tab switchs
    ([#269](https://github.com/shortcuts/no-neck-pain.nvim/issues/269))
    ([2afc30d](https://github.com/shortcuts/no-neck-pain.nvim/commit/2afc30d8ec23000c4f830310661f63ee3b581a42))

    ---
    This PR was generated with [Release
    Please](https://github.com/googleapis/release-please). See
    [documentation](https://github.com/googleapis/release-please#release-please).

    Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

**Comment by shortcuts** [comment]
Hey @timtyrrell, thanks for reporting! are the steps described here https://github.com/shortcuts/no-neck-pain.nvim/blob/main/tests/test_tabs.lua#L260-L419 similar to what you are doing?

**Comment by timtyrrell** [comment]
I can trigger it every time without any trouble moving from tab to tab and running my toggle key `<Leader>nn`? Let me know if you can't and I can look harder but I am not doing anything differently than the example above

**Comment by shortcuts** [comment]
oh ok I got confused in step 2 it seems (or I was too tired 🤔) now I get the issue, checking for a fix!

**Comment by shortcuts** [comment]
(fun fact, there was 2 bugs in fact)

**Comment by timtyrrell** [comment]
Hey, good job fixing one of them then 🏆

**Comment by shortcuts** [comment]
hey @timtyrrell, this is fixed (for real now), in [the 1.6.2 version](https://github.com/shortcuts/no-neck-pain.nvim/releases/tag/v1.6.2)!

**Comment by timtyrrell** [comment]
Yup, that fixes it. I have one more coming in
